bin/prettier.js

```diff
@@ -193,7 +191,11 @@ if (stdin) {
       if (write) {
         // Don't write the file if it won't change in order not to invalidate
         // mtime based caches.
-        if (output !== input) {
+        if (output === input) {
+          console.log("[ignore] %s", filename);
```

Comment:
Maybe `[no change]`, then? That also has the benefit of being a different length, which I've found helps me scan a column of statuses.